tests/main/interfaces-gnome-online-accounts-service/task.yaml

```diff
@@ -0,0 +1,55 @@
+summary: Ensure that the gnome-online-accounts-service interface works
+
+# Only test on classic systems with AppArmor DBus mediation
+systems: [ ubuntu-1* ]
+
+environment:
+    XDG_CONFIG_HOME: $(pwd)/config
+    XDG_DATA_HOME: $(pwd)/share
+    XDG_CACHE_HOME: $(pwd)/cache
+
+prepare: |
+    . $TESTSLIB/pkgdb.sh
+    echo "Ensure we have a working goa-daemon"
+    distro_install_package gnome-online-accounts
+    snap install --edge test-snapd-gnome-online-accounts
```

Comment:
It would be nice if this didn't have to be installed over the network, but I guess there is currently no way to create a snap with compiled code in tests/lib/snaps....